test/pages/mailbox.page.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-undef */
+/* eslint-disable linebreak-style */
+/* eslint-disable class-methods-use-this */
+
+const Page = require('./page');
+
+class MailboxPage extends Page {
+  get loginButton() { return $('.//*[@id="ybar-inner-wrap"]/div[3]/div/div[3]/div[1]/div/a'); }

```

Comment:
The logic of using the Page Object pattern is to collect all selectors for 1 page in 1 file.
For example, you should create login.page.js, create.email.page.js and etc files with locators and methods specified for these pages